docs/diagnostics.md

```
@@ -113,6 +113,59 @@ emitted at `information` severity.
+in a forward directive (`@lsp-do`, `@lsp-run`, `@lsp-include`) or
+backward header directive (`@lsp-done-by`, `@lsp-run-by`,
+`@lsp-included-by`) — resolves to an on-disk file that has different
+letter casing. The file is found and symbols are inherited (no
+undefined-symbol cascade), but the diagnostic asks you to fix the
+spelling.
+
+**Forward message** (`do`/`run`/`include` commands and forward
+directives): notes that Stata will not find the file on case-sensitive
+filesystems and shows both spellings:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)